Constitution.txt

```diff
@@ -1,490 +1,264 @@
+6: The Senate shall have the sole Power to try all Impeachments. When sitting for that Purpose, they shall be on Oath or Affirmation. When the President of these United States is tried, the Chief Justice shall preside: And no Person shall be convicted without the Concurrence of two thirds of the Members present.
 
-7: Judgment in Cases of impeachment shall not extend further than to removal from Office, and disqualification to hold and enjoy any Office of honor, Trust or Profit under the United States: but the Party convicted shall nevertheless be liable and subject to Indictment, Trial, Judgment and Punishment, according to Law.
+7: Judgment in Cases of impeachment shall not extend further than to removal from Office, and disqualification to hold and enjoy any Office of honor, Trust or Profit under these United States: but the Party convicted shall nevertheless be liable and subject to Indictment, Trial, Judgment and Punishment, according to Law.
 
 Section 4
-1: The Times, Places and Manner of holding Elections for Senators and Representatives, shall be prescribed in each State by the Legislature thereof; but the Congress may at any time by Law make or alter such Regulations, except as to the Places of chusing Senators.
+1: The Times, Places and Manner of holding Elections for Representatives, and for deciding the Appointments of Senators, shall be prescribed in each State by the Constitution thereof, but such Elections must occur every second Year from the date on which the State joins the Union.
 
-2: The Congress shall assemble at least once in every Year, and such Meeting shall be on the first Monday in December,5 unless they shall by Law appoint a different Day.
+2: The Congress shall assemble at least once in every Year, and such Meeting shall be on the first Monday in December, unless they shall by Law appoint a different Day. Neither house shall assemble for a period of more than one Hundred days in a year, except as may be necessary in time of emergency which threatens the immediate physical safety of the Citizens of any State.
 
 Section 5
 1: Each House shall be the Judge of the Elections, Returns and Qualifications of its own Members, and a Majority of each shall constitute a Quorum to do Business; but a smaller Number may adjourn from day to day, and may be authorized to compel the Attendance of absent Members, in such Manner, and under such Penalties as each House may provide.
 
-2: Each House may determine the Rules of its Proceedings, punish its Members for disorderly Behaviour, and, with the Concurrence of two thirds, expel a Member.
+2: Each House may determine the Rules of its Proceedings, punish its Members for disorderly Behavior, and, with the Concurrence of two thirds, expel a Member, who shall when so expelled be ineligible to return until one full term of Representation has elapsed.
```

Comment:
This is an example of several "cooling off periods" I have built into these updates. In this case, it actually grants some more authority to the House than the original Constitution did, because it lets them keep someone away for a substantial period of time. If someone is ejected but massively supported by their local community, they can easily nominate a surrogate who will carry out similar actions, but this helps to concentrate the attention on their ideas, rather than on their person.